### PR TITLE
Explain --net=host option usage

### PR DESCRIPTION
**Pull request by ankushagarwal**
Signed-off-by: Ankush Agarwal ankushagarwal11@gmail.com

### CODE REVIEW

**Comment by moxiegirl** [comment]
@ankushagarwal Why is the performance of the host mode better than the bridge?  I always like to add why if I can to the docs.  Helps people to make the choice.   If you don't have an explanation for that. No worries.  I'll LGTM without it.

**Comment by SvenDowideit** [comment]
LGTM

File: docs/sources/reference/run.md [hunk 1]
```diff
@@ -279,6 +279,13 @@ container.  The container's hostname will match the hostname on the host
 system.  Publishing ports and linking to other containers will not work
 when sharing the host's network stack.
 
+Compared to the default `bridge` mode, the `host` mode gives *significantly*
+better networking performance since it uses the host's native networking stack
+wheras the bridge has to go through one level of virtualizaion through the
+docker daemon. It is recommended to run containers in this mode when their
+networking performance in critical, for example a production Load Balancer
```
Comment (thaJeztah) [inline docs/sources/reference/run.md]: Typo: "in critical" -> "is critical"

comma after "for example"

File: docs/sources/reference/run.md [hunk 2]
```diff
@@ -279,6 +279,13 @@ container.  The container's hostname will match the hostname on the host
 system.  Publishing ports and linking to other containers will not work
 when sharing the host's network stack.
 
+Compared to the default `bridge` mode, the `host` mode gives *significantly*
+better networking performance since it uses the host's native networking stack
+wheras the bridge has to go through one level of virtualizaion through the
+docker daemon. It is recommended to run containers in this mode when their
```
Comment (thaJeztah) [inline docs/sources/reference/run.md]: "when their" -> "if their" (I think)

File: docs/sources/reference/run.md (hunk 1 above)
Comment (ankushagarwal) [inline docs/sources/reference/run.md]: ✅

File: docs/sources/reference/run.md (hunk 2 above)
Comment (ankushagarwal) [inline docs/sources/reference/run.md]: I am not sure about this one, so I am leaving it as is.

**Pull request merged**